bullet_train-api/.gitignore

```diff
@@ -8,7 +8,6 @@
 /test/dummy/log/*.log
 /test/dummy/storage/
 /test/dummy/tmp/
-Gemfile.lock
```

Comment:
[Bundler recommends including `Gemfile.lock` in the repo of gems.](https://bundler.io/guides/faq.html#using-gemfiles-inside-gems)